packages/trader/src/Modules/Contract/Containers/contract-replay.jsx

```diff
@@ -83,7 +82,7 @@ const ContractReplay = ({
     );
 
     const unsupportedContractOnConfirm = () => {
-        this.props.history.push('/reports');
+        history.push('/reports/statement');
```

Comment:
Should we navigate to the previous screen?
User can visit `contract-replay` from profit-table too. In that case, it would be good to navigate them to profit table

Reply:
Good point thanks. Fixed.